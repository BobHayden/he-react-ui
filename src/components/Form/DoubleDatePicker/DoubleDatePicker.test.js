import createTestContext from 'react-cosmos-test/enzyme';
import fixtures from './DoubleDatePicker.fixtures';

fixtures.forEach(fixture => {
  const context = createTestContext({ fixture });
  context.mount();
  test(`<DoubleDatePicker /> rendered correctly with ${
    fixture.name
  } fixture`, () => expect(context.getWrapper()).toMatchSnapshot());
});
