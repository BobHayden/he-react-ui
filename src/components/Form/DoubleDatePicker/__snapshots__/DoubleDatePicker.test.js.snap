// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DoubleDatePicker /> rendered correctly with base fixture 1`] = `
<DoubleDatePicker>
  <div
    className="main"
  />
</DoubleDatePicker>
`;
